.ruby-version

```diff
@@ -1 +1 @@
-2.7.4
+3.0.0
```

Comment:
We tried to support the latest version of Ruby, but ran into an issue c15cb3c9b3f924af3bbb7aca56ffd7f40dc836ee